creator-node/src/middlewares.js

```diff
@@ -197,6 +197,11 @@ async function ensurePrimaryMiddleware(req, res, next) {
 
 /** Blocks writes if node has used over `maxStorageUsedPercent` of its capacity. */
 async function ensureStorageMiddleware(req, res, next) {
+  // Force wipe syncs will free up storage space so we want to perform them regardless of current usage
+  if (req.body?.forceWipe) {
```

Comment:
i think we should do this as a one-off check in https://github.com/AudiusProject/audius-protocol/blob/28811d981a8c37fad46d6441db5c338b00ce9335/creator-node/src/components/replicaSet/replicaSetController.js#L319.

for example
```
router.post(
  '/sync',
  (req, res, next) => {
    if (req.body?.forceWipe) return next()
    else return ensureStorageMiddleware()
  },
  handleResponse(syncRouteController)
)
```

Reply:
makes sense

Reply:
changed and re-tested. how does it look now?